terminus/geometry/latlon.py

```diff
@@ -64,55 +65,71 @@ def __repr__(self):
     def sum(self, other):
         return LatLon(self.lat + other.lat, self.lon + other.lon)
 
+    def curvature(self):
+        # Radius of planet curvature (meters)
+        curvature = Ecef.equatorial_radius() / math.sqrt(1 - Ecef.first_eccentricity_parameter() ** 2 * self._sin_lat() ** 2)
+        return curvature
+
+    def latlon_to_ecef(self):
+        x = self.curvature() * self._cos_lat() * self._cos_lon()
+        y = self.curvature() * self._cos_lat() * self._sin_lon()
+        z = ((Ecef.polar_radius() ** 2) / (Ecef.equatorial_radius() ** 2)) * self.curvature() * self._sin_lat()
+        return Ecef(x, y, z)
+
+    def global_to_ecef_rotation(self, enu):
```

Comment:
Both take a LatLon parameter, cause both depend on the origin point, which is given in LatLon. I chose to do it this way (having self a LatLon object) to enforce the idea of this depending on that origin. I mean, for each LatLon point, I have a conversion from ecef to global and from global to ecef. Tell me if you want me to change it to make the LatLon objet be a parameter instead of being the object asked for the transformation or if I should leave it this way.